src/controllers/recipe.controller.ts

```diff
@@ -24,11 +24,13 @@ export class RecipeController {
         userId: _id,
       };
 
-      const savedRecipe = (
-        await recipeService.createRecipe(recipeData)
-      ).toJSON();
+      const savedRecipe = await recipeService.createRecipe(recipeData);
 
-      webhookService.sendWebhookNotification(_id, "create_recipe", savedRecipe);
+      await webhookService.sendWebhookNotification(
```

Comment:
Here is an interesting part. Should we wait for webhook event to be sent? Or can we not wait for it at all? From this request perspective we don't care if webhook events were sent or not. So we could resign from `await` here. But perhaps in our architecture design we don't allow such things? It's up to you. Also if you decide to not await this, you have to wrap this in additional try catch or make sure it never throws an error. Otherwise catch block will be triggered and we'll get "can't set headers already sent" error.